docs/django-widget.rst

```diff
@@ -9,32 +9,63 @@ Django Widget
 Settings
 --------
 
-Besides required ``pub_key``, ``secret`` settings there are optional settings,
-for example, ``widget_version`` or ``widget_build``:
+Besides required ``pub_key``, ``secret`` settings there are optional settings.
+
+Below is the default config:
 
 .. code-block:: python
 
-    UPLOADCARE = {
-        'pub_key': 'demopublickey',
-        'secret': 'demoprivatekey',
-        'widget_version': '3.x',  // ~= 3.0 (latest)
-        'widget_build': 'min',  // without jQuery
-        'cdn_base': 'https://cdn.mycompany.com',
+   UPLOADCARE = {
+        "pub_key": "",
+        "secret": "",
+        "cdn_base": None,
+        "upload_base_url": None,
+        "use_legacy_widget": False,
+        "use_hosted_assets": True,
+        "legacy_widget": {
+            "version": "3.x",
+            "build": "full.min",
+            "override_js_url": None,
+        },
+        "widget": {
+            "version": "0.x",
+            "variant": "regular",
+            "build": "min",
+            "options": {},
+            "override_js_url": None,
+            "override_css_url": {
+                "regular": None,
+                "inline": None,
+                "minimal": None,
+            },
+        },
```

Comment:
I haven't looked through everywhere, but is it obvious for devs to find those keys, bases, etc? Should it be mentioned in README, or maybe add some comments with relevant URLs here?

Reply:
Good point. Yes, it's missing here.

But there's a section in the [Quickstart](https://pyuploadcare.readthedocs.io/en/latest/quickstart.html) on about how to obtain it.

Reply:
If I learned anything is that people sometimes don't read docs. :-) But maybe you're right.

Reply:
Should probably copy-paste that section.

Reply:
...and point out that `pub_key` and `secret` are the only things that are required, everything else is optional.

Reply:
Fixed in 192d10bdf3e2588b3fe6fbe4ed5434dda5426a99